build.gradle.kts

```diff
@@ -0,0 +1,44 @@
+buildscript {
+    repositories {
+        mavenCentral()
+        google()
+        jcenter()
+        gradlePluginPortal()
+        maven { url = uri( "https://dl.bintray.com/kotlin/kotlinx") }
+        maven { url = uri("https://dl.bintray.com/kotlin/kotlin-eap") }
+        maven { url = uri("https://dl.bintray.com/jetbrains/kotlin-native-dependencies") }
+    }
+
+    dependencies {
+        classpath(com.codingfeline.buildkonfig.build.Dependencies.kotlinPlugin)
+        classpath(com.codingfeline.buildkonfig.build.Dependencies.dokkaPlugin)
+        classpath(com.codingfeline.buildkonfig.build.Dependencies.gradleVersionsPlugin)
+        classpath(com.codingfeline.buildkonfig.build.Dependencies.pluginPublishPlugin)
+    }
+}
+
+plugins {
+    id("com.github.ben-manes.versions") version com.codingfeline.buildkonfig.build.Versions.benManesVersionsPlugin
```

Comment:
Oops, didn't commit the file. I will do it later. Sorry for that.

Reply:
Probably that's because you named the package `build`.
A directory named `build` is ignored by `.gitignore`, so we need to come up with better package name.

Reply:
So, I renamed the package to `buildsrc` and committed the missing file. Not the best name, but it makes clear that it's part of the build logic.